learning.py

```diff
@@ -655,7 +655,7 @@ def predict(example):
 
 
 def NeuralNetLearner(dataset, hidden_layer_sizes=[3],
-                     learning_rate=0.01, epochs=100, activation = sigmoid):
+                     learning_rate=0.01, epochs=100, activation = sigmoid,momentum=False):
```

Comment:
To comply with the overall repo style, add a space after commas.

Reply:
spaces added